support-frontend/cloud-formation/cfn.yaml

```diff
@@ -545,3 +545,30 @@ Resources:
       TreatMissingData: notBreaching
     DependsOn:
     - StateMachineUnavailableMetric
+
+  ServerSideCreateFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+      AlarmName: !Join
+        - ' '
+        - - !FindInMap [ Constants, Alarm, Urgent ]
+          - !Ref 'Stage'
+          - 'support-frontend create subscription call failed'
+      AlarmDescription: !Join
+        - ' '
+        - - 'Impact - Someone pressed buy on a subscription but received an error.'
+          - !FindInMap [ Constants, Alarm, Process ]
+      MetricName: ServerSideCreateFailure
+      Namespace: support-frontend
+      Dimensions:
+        - Name: Stage
+          Value: PROD
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Threshold: 1
+      Period: 60
+      EvaluationPeriods: 1
```

Comment:
this is the same as support-workers failure in prod, and should probably be reacted to similarly, in fact it's even more likely to be a "real" issue as test users shouldn't trigger it.